go/chat/keyfinder.go

```diff
@@ -26,27 +26,34 @@ func (k *KeyFinder) cacheKey(tlfName string, tlfPublic bool) string {
 
 // find finds keybase1.TLFCryptKeys for tlfName, checking for existing
 // results.
-func (k *KeyFinder) Find(ctx context.Context, tlf keybase1.TlfInterface, tlfName string, tlfPublic bool) (keybase1.TLFCryptKeys, error) {
+func (k *KeyFinder) Find(ctx context.Context, tlf keybase1.TlfInterface, tlfName string, tlfPublic bool) (keybase1.GetTLFCryptKeysRes, error) {
 	ckey := k.cacheKey(tlfName, tlfPublic)
 	existing, ok := k.keys[ckey]
 	if ok {
 		return existing, nil
 	}
 
-	var keys keybase1.TLFCryptKeys
+	var keys keybase1.GetTLFCryptKeysRes
 	if tlfPublic {
-		cid, err := tlf.PublicCanonicalTLFNameAndID(ctx, tlfName)
+		res, err := tlf.PublicCanonicalTLFNameAndID(ctx, keybase1.PublicCanonicalTLFNameAndIDArg{
+			TlfName:          tlfName,
+			IdentifyBehavior: keybase1.TLFIdentifyBehavior_CHAT,
+		})
 		if err != nil {
-			return keybase1.TLFCryptKeys{}, err
+			return keybase1.GetTLFCryptKeysRes{}, err
 		}
-		keys.CanonicalName = cid.CanonicalName
-		keys.TlfID = cid.TlfID
+		keys.Breaks = res.Breaks
```

Comment:
Copying a struct over field-by-field usually shouldn't be necessary.  Maybe these two data structures should be one in the same?